src/services/goals.js

```diff
@@ -1248,7 +1248,9 @@ export async function createOrUpdateGoals(goals) {
         }
 
         await objective.update({
-          title,
+          ...(!objective.dataValues.onApprovedAR
+            && title.trim() !== objective.dataValues.title.trim()
```

Comment:
Could you add a test? It looks like you could add another test to ```services/createOrUpdateGoalsForActivityReport.test.js``` and follow the pattern there